docs/source/2018Michael.rst

```diff
@@ -68,6 +68,29 @@ Model Configuration and Datasets
     .. code-block:: bash
    
       crontab Michael.crontab     
+  .. group-tab:: SRW.v1.0
+
+    The case runs are initialized at 00z Oct 09, 2018 with 90 hours forecasting. The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, cycle hour, and fcst length are listed below.
```

Comment:
This change is also needed in the MRW section
```suggestion
    The case was initialized at 00z Oct 09, 2018 and forecast out to 90 hours. The app uses ``config.sh`` to define the runtime settings. The settings that need to be modified to set up the first cycle, last cycle, cycle hour, and fcst length are listed below.
```

Reply:
Changed, thanks!